src/index.js

```diff
@@ -1,33 +1,35 @@
 import './style.css';
+import {
+  tasks,
+  addTask,
+  editTask,
+} from './handleTask.js';
 
 const listContents = document.querySelector('.listContents');
+const refresh = document.getElementById('refresh');
+const submit = document.querySelector('.submit');
+const formData = document.getElementById('text');
 
-const data = [
-  {
-    description: 'ghislain Developer',
-    completed: true,
-    index: 0,
-  },
-  {
-    description: 'Mitahi hacker',
-    completed: true,
-    index: 1,
-  },
-  {
-    description: 'zazu security',
-    completed: true,
-    index: 2,
-  },
-];
+refresh.addEventListener('click', () => {
+  document.location.reload();
+});
```

Comment:
As suggested by the previous Code Reviewer, Please refrain from using document.location.reload( ). Too many reload of the page are considered bad UX and make the page slow and laggy (especially if you get data from a backend API). Instead, you can remove the specific node from the DOM by first selecting it using querySelector and then removing it using document.removeChild(). Even adding and editing can be done in similar ways. Please remove all the document.location.reload() commands from your Application.